Makefile

```diff
@@ -290,12 +329,19 @@ GOPATH=$(shell go env GOPATH)
 	go get github.com/golang/mock/mockgen
 	cd "${GOPATH}/src/github.com/golang/mock/mockgen" && git checkout 1.3.1 && go get ./... && go install ./... && cd -
 	go get golang.org/x/tools/cmd/goimports
-	go get github.com/fzipp/gocyclo/cmd/gocyclo
+	GO111MODULE=on go get github.com/fzipp/gocyclo/cmd/gocyclo@v0.3.1
```

Comment:
curious where the v0.3.1 comes from -- is this the go modules enabled version?

Reply:
This is the version we're actually currently using in agent for get-deps, so went with it for consistency. I had to change this due to receiving the following error:
```
package io/fs: unrecognized import path "io/fs": import path does not begin with hostname
```
Based on [this](https://github.com/tdewolff/minify/issues/388) github issue, looks like this is caused by the latest version of that package requiring at least go 16.